src/assets/arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I suggest putting all these assets for the home page in their folder. So put it in src/assests/home/